File: Groger/Groger.Entity/Grocery.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Groger.Entity
{
    public class Grocery
    {
        [Required]
        public int Id { get; set; }
        [Required]
        [StringLength(100, MinimumLength = 6)]
        public string Name { get; set; }
        [Required]
        [StringLength(100, MinimumLength = 6)]
        public string Description { get; set; }
        [Url]
        public string Picture { get; set; }
        
        public int? CategoryId { get; set; }
        public virtual Category Category { get; set; }

        public virtual ICollection<ClusterGrocery> ClusterGroceries { get; set; }
    }
}

File: GrogerApp/app/src/main/java/com/groger/grogerapp/service/model/Grocery.kt
package com.groger.grogerapp.service.model

import java.io.Serializable

data class Grocery(
        val id: Int,
        val name: String,
        val description: String,
        // Not present in grocery, only in clusterGrocery
        val quantity: Int?,
        val picture: String?,
        val category: String?
)  : Serializable